script.js

```diff
@@ -1,57 +1,83 @@
-const add = document.getElementById('add');
-const awesomeBooks = [];
+const StoredBooks = [];
 
-add.addEventListener('click', (e) => {
-  e.preventDefault();
-  const title = document.getElementById('title');
-  const author = document.getElementById('author');
+class Display {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 
-  const book = {
-    Title: title.value,
-    Author: author.value,
-  };
-  const list = document.getElementById('book-list');
-  const newDiv = document.createElement('div');
-  newDiv.classList.add('book');
+  static addtolist() {
+    const Title = document.getElementById('title').value;
+    const Author = document.getElementById('author').value;
+    const list = document.getElementById('book-list');
+    const newDiv = document.createElement('div');
+    newDiv.classList.add('book');
+    const newbook = new Display(Title, Author);
+    StoredBooks.push(newbook);
+
+    newDiv.innerHTML += `
+    <span>${newbook.title}</span><span> by</span><span> ${newbook.author}</span>
+      <button class="delete">Delete</button>
+    `;
+
+    list.appendChild(newDiv);
+    document.getElementById('title').value = '';
+    document.getElementById('author').value = '';
+  }
+
+  static addtolocal() {
+    localStorage.setItem('books', JSON.stringify(StoredBooks));
+  }
+
+  static delBook(e) {
+    e.preventDefault();
+    if (e.target.classList.contains('delete')) {
+      e.target.parentElement.remove();
+    }
+  }
 
-  awesomeBooks.push(book);
+  static removelocal(e) {
+    StoredBooks.forEach((newbook, i) => {
+      if (
+        e.target.parentElement.firstElementChild.textContent === newbook.title
+      ) {
+        StoredBooks.splice(i, 1);
+      }
+    });
+    localStorage.setItem('books', JSON.stringify(StoredBooks));
+  }
```

Comment:
- [ ] When I add two books that have the same title, and I try to delete one, the two of them are removed from the localStorage because of this class method. It would be nice to use to differentiate each book in the array even though they have the same title or you can use the index to uniquely remove the targeted book. 

`this.books = this.books.filter(({ id }) => bookID !== id);`

#### Kindly fix this error. 👍